workflows/ampliseq.nf

```diff
@@ -364,7 +388,7 @@ workflow AMPLISEQ {
         FORMAT_FASTAINPUT( ch_input_fasta )
         ch_unfiltered_fasta = FORMAT_FASTAINPUT.out.fasta
     } else {
-        ch_unfiltered_fasta = DADA2_MERGE.out.fasta
+        ch_unfiltered_fasta = ch_dada2_fasta
```

Comment:
The lines
```
    if ( params.input_fasta ) {
        FORMAT_FASTAINPUT( ch_input_fasta )
        ch_unfiltered_fasta = FORMAT_FASTAINPUT.out.fasta
    } else {
        ch_unfiltered_fasta = DADA2_MERGE.out.fasta
        ch_unfiltered_fasta = ch_dada2_fasta
    }
```
are actually misplaced (maybe by me?) under the headline `Modules : Filter rRNA`. This part isnt filtering rRNA but allowing the input of a ASV fasta by `--input_fasta <file>`. I assume ASV clustering doesnt need to be made on `--input_fasta`, so thats fine that this block is after the clustering step.
No action required, just fyi.